Source/Make.WarpX

```diff
@@ -5,6 +5,13 @@ OPENBC_HOME ?= ../../../openbc_poisson
 USE_MPI       = TRUE
 USE_PARTICLES = TRUE
 
+ifeq ($(USE_GPU),TRUE)
+  USE_OMP  = FALSE
+  USE_CUDA = TRUE
+  USE_ACC  = TRUE
+  NVCC_HOST_COMP = gcc
```

Comment:
Out of curiosity, why do we need the above line?
(`NVCC_HOST_COMP = gcc`)
Is still being used even when we use the `pgi` compiler?

Reply:
Yes, we actually use gcc to compile the host code, even when we use pgi to compile the device code. This was to work around limitations pgi had with some C++11 features we use, the details of which I'm not remembering right now.